src/blog/2025/05/displaying-embeded-webpages-on-node-red-dashboard.md

```diff
@@ -0,0 +1,134 @@
+  
+- **Quick Decisions:** Having all your key data together helps you see what’s important at a glance and act faster.
+
+- **Cut Down on Clicks:** The fewer actions needed to get to your information, the more time you can spend on actually getting things done.
+
+## How to Embed Webpages On your FlowFuse Dashboard
+
+Embedding external content into your FlowFuse dashboard is straightforward and flexible. In this section, you'll learn two ways to do it: using a direct URL and using embed code. We’ll also cover common issues you might face and how to fix them. Lastly, you'll see how to embed one FlowFuse dashboard inside another, making it easier to centralize important views in one place.
+
+### Prerequisites
+
+Before you begin embedding webpages on FlowFuse Dashboard, make sure you have the following:
+
+- **Running FlowFuse Instance:** Make sure you have a FlowFuse instance set up and running. If you don't have an account, check out our [free trial](https://app.flowfuse.com/account/create).
+- **FlowFuse Dashboard:** Ensure you have [FlowFuse Dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) (also known as Node-RED Dashboard 2.0 in the community) installed and properly configured on your instance.
+- **@flowfuse/node-red-dashboard-2-ui-iframe:** Ensure you have [node-red-dashboard-2-ui-iframe](https://flows.nodered.org/node/@flowfuse/node-red-dashboard-2-ui-iframe) installed.
+
+#### Step 1: Obtain the URL or Embed Code of the Webpage You Want to Embed
+
+The first step is identifying and grabbing the URL of the webpage or external content you want to embed into your FlowFuse dashboard. This could be a live dashboard, report, Google Maps, video, or any other type of content.
+
+When embedding content from third-party sources, make sure the following conditions the page or content you're embedding should be publicly accessible or have the proper permissions for embedding. For example, private reports or webpages require login credentials or an API key, which should be handled securely.
+
+Some websites may restrict embedding through iframes due to [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/CORS) policies. Check whether the external site allows embedding if the content doesn’t load properly.
+
+Many platforms, such as YouTube and Google Maps, provide a specific "embed code or URL" that is more suitable for embedding. Ensure you use the correct embed link or code these platforms offer to guarantee smooth integration.
+
+Once the URL or embed code is ready, move on to the next step.
+
+#### Step 2: Embedding the Webpage on FlowFuse Dashboard
+
+Now that you’ve gathered the URL or embed code for the external content you want to embed, it’s time to add it to your FlowFuse dashboard. Below are two methods for embedding external content:
+
+##### 2.1 Embed via URL 
```

Comment:
Can you add a screenshot here of the config for the node please? Feels a little too "blackbox" atm